### PR TITLE
Fixing jquery for bower

### PR DESCRIPTION
**Pull request by katafira**
<h1>Purpose</h1>
This configuration fixes bower and patternfly issues. <h3>This parts from PR #37, so it has some files you can also find in it.</h3> Merging is supposed to solve said duplication, which was needed for testing if this solution works.

Ready to review. 
@sergio-ocon @delaluzparra @alexvkcr

### CODE REVIEW

**Comment by sergio-ocon** [comment]
This has been fixed in 
https://github.com/propedeutica/shift-reservation/pull/1

And thus is correct in that version. Please have a look at it.

**Comment by sergio-ocon** [comment]
It seems that as the JS is loaded into the asset pipeline, you only need to add the links into application.js, and they are available automatically
You can find them in the asset pipeline doing in the console:
`Rails.application.assets.find_asset 'Rails.application.assets.find_asset 'bootstrap/dist/js/bootstrap.min.js'`

**Comment by katafira** [comment]
@agarciavera @sebjimenez

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -13,13 +13,10 @@
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
-
-//
+//= require_tree .
```
Comment (sergio-ocon) [inline app/assets/javascripts/application.js]: Not needed, delete it

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -8,11 +8,21 @@
...
       <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-
+        
     <title><%=full_title(yield(:title))%></title>
 
     <%= render 'layouts/shim' %>
-
+                  
+                  <!-- PatternFly Styles -->
+            <link href="bower_components/patternfly/dist/css/patternfly.min.css" rel="stylesheet" media="screen, print">
```
Comment (sergio-ocon) [inline app/views/layouts/application.html.erb]: Not needed, it is already loaded if you configure it properly

**Comment by alexvkcr** [comment]
Looks good to me, tried locally @sergio-ocon

File: config/routes.rb [hunk 3]
```diff
@@ -10,12 +10,13 @@
     root to: "static_pages#home"
     get 'home' => 'static_pages#home'
     resources :users, only: [:show, :index, :destroy]
-    resources :offsprings, only: [:new, :create, :destroy]
+    resources :offsprings, only: [:new, :index, :create, :destroy]
```
Comment (sergio-ocon) [inline config/routes.rb]: With this route, any user can see the index, and that should not be possible
resources :offspring, only: [:index], of: user.admin?
Or similar

**Comment by sergio-ocon** [comment]
Please fix conflicts in the PR

**Comment by katafira** [comment]
@sergio-ocon  Fixed conflicts.

**Pull request merged**